src/kaltura-live-plugin.tsx

```diff
@@ -106,6 +109,10 @@ export class KalturaLivePlugin extends KalturaPlayer.core.BasePlugin implements
     this.updateLiveStatus();
   };
 
+  private _getOfflineSlateUrl = () => {
+    return this.config.offlineSlateUrl || this.player.poster;
```

Comment:
what if theres no poster or if poster fails to load ?

Reply:
if URL invalid or poster doesn't exist user see black background as default slate (like error slate has)
<img width="1012" alt="image" src="https://github.com/kaltura/playkit-js-kaltura-live/assets/51074448/d2bd439a-5751-454f-9276-1c7a9f2c410e">